content/en/docs/concepts/services-networking/service.md

```diff
@@ -428,8 +428,7 @@ variables and DNS.
 
 When a Pod is run on a Node, the kubelet adds a set of environment variables
 for each active Service.  It supports both [Docker links
-compatible](https://docs.docker.com/userguide/dockerlinks/) variables (see
-[makeLinkVariables](https://releases.k8s.io/{{< param "fullversion" >}}/pkg/kubelet/envvars/envvars.go#L49))
+compatible](https://docs.docker.com/userguide/dockerlinks/) variables (see [makeLinkVariables](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/envvars/envvars.go#L72))
```

Comment:
In future, if the `envvars.go` file is updated, the link to the L72 might point the line other than `makeLinkVariables` function.

One possible solution is to use permalink: https://github.com/kubernetes/kubernetes/blob/95390e6476fca15b0f55cde2c142ec18ca57aee0/pkg/kubelet/envvars/envvars.go#L72

But if the permalink is used, then the link will point the `95390e6` version of `envvars.go` file, not the latest version.

Hmm.. I think that for now, the @victuos 's suggestion will be okay. 😊

Reply:
Point this to a tag please. The 'master' branch is changing every day.

Reply:
I agree
/lgtm cancel

Reply:
@jihoon-seo @tengqm @sftim is it okay now?

Reply:
@victuos ,  I don't see the link pointing to a tag as @tengqm suggested and @sftim agreed.
Please update this PR to point to a tag instead of a line number as the k/website main branch changes daily and the link will be incorrect eventually

Reply:
@tengqm @sftim @reylejano I thought I did it correctly:
https://github.com/kubernetes/kubernetes/blob/dd2d12f6dc0e654c15d5db57a5f9f6ba61192726/pkg/kubelet/envvars/envvars.go#L72
Where dd2d... is the tag which will always point to the same line since the tag is fixed, or am I wrong? Could you clear up my confusion if necessary?

Reply:
`dd2d12f6dc0e654c15d5db57a5f9f6ba61192726` is a ref but not a tag. Anyway, it'll work.